### PR TITLE
Added support for multiple authors for blog posts

### PR DESCRIPTION
**Pull request by Fgerthoffert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,16 +73,26 @@ teaser:
...
 
-We are using `kramdown` markdown converter. You can find out more about it here [kramdown](http://kramdown.gettalong.org). We highly recommend going through its syntax documentation 
-to get yourself more familiar with the coding style. 
+We are using `kramdown` markdown converter. You can find out more about it here [kramdown](http://kramdown.gettalong.org). We highly recommend going through its syntax documentation
+to get yourself more familiar with the coding style.
+
+#### Support for multiple authors
+
+Replace 'author' in the blog post with an array of authors like this:
+```
```
Comment (btiernay) [inline README.md]: Does "```yaml" work?

File: README.md [hunk 2]
```diff
@@ -73,16 +73,26 @@ teaser:
...
+authors:
+    - firstname_lastname
```
Comment (btiernay) [inline README.md]: Really wished we used the `firstname-lastname` format. Looks nicer and also http://stackoverflow.com/questions/119312/urls-dash-vs-underscore

File: _layouts/category.html [hunk 3]
```diff
@@ -17,57 +18,77 @@
...
+                    {% for currentAuthor in post.authors %}
+                      {% assign author = site.data.authors[currentAuthor] %}
+                      <a href="{{ site.baseurl }}/blog/category/{{ currentAuthor }}"><strong>{{ author.name }}</strong></a>{% unless forloop.last %}, {% endunless %}
+                    {% endfor %}
                     </p>
+                  {% else if authorCount > 0 %}
+                    {% assign author = site.data.authors[post.author] %}
+                    <p class="post-author">By: <a href="{{site.baseurl}}/blog/category/{{post.author}}"><strong>{{ author.name }}</strong></a></p>
+                  {% endif %}
+                  <div class="circles">
```
Comment (btiernay) [inline _layouts/category.html:40]: I guess these are some circles.

File: _layouts/post.html [hunk 4]
```diff
@@ -63,37 +74,68 @@ <h3 class="post-title"><a href="{{ site.baseurl }}{{ page.url }}">{{ page.title
...
-            <span>
-              {{author.position}}
-            </span>
+    {% if authorsCount > 0 %}
+      {% for currentAuthor in page.authors %}
+        {% assign author = site.data.authors[currentAuthor] %}
+        <div class="author-box columns">
+          <div class="author">
+            <div class="small-2 columns">
+              <img src="{% if author[1].avatar contains 'http' %}{{ author[1].avatar }}{% else %} {{site.urlimg}}avatars/{{author.avatar}} {% endif %}" title="{{ author.name }}"/>
```
Comment (btiernay) [inline _layouts/post.html]: What's this shizzle?

File: _layouts/category.html (hunk 3 above)
Comment (Fgerthoffert) [reply _layouts/category.html:40]: 🎱 not changed by the PR, those circles were there before

File: _layouts/post.html (hunk 4 above)
Comment (Fgerthoffert) [reply _layouts/post.html]: not sure, but it's not coming from this PR

File: README.md (hunk 2 above)
Comment (Fgerthoffert) [reply README.md]: What about people with a dash in their firstname (phuong-my for example) ?

File: README.md (hunk 1 above)
Comment (Fgerthoffert) [reply README.md]: 👍

File: _layouts/post.html [hunk 5]
```diff
@@ -63,37 +74,68 @@ <h3 class="post-title"><a href="{{ site.baseurl }}{{ page.url }}">{{ page.title
...
+                <a class="author-blogs" href="{{ site.baseurl }}/blog/category/{{ currentAuthor }}">Blog Posts »</a>
+                <a class="author-github" href="https://github.com/{{ author.github }}"> GitHub </a>
+              </div>
+            </div>
           </div>
-          <div class="author-info">
-            <span>{{ author.info }}</span>
+        </div>
+      {% endfor %}
+    {% else if authorCount > 0 %}
```
Comment (shanewilson) [inline _layouts/post.html]: isn't this is the same condition as above? `authorCount > 0`

Comment (shanewilson) [reply _layouts/post.html]: oh authors vs author

Comment (shanewilson) [reply _layouts/post.html]: could you do something like `authors.concat(author)` and then just loop over `authors`?

Comment (Fgerthoffert) [reply _layouts/post.html]: 👍  done, code is simpler now

File: README.md [hunk 6]
```diff
@@ -73,16 +73,26 @@ teaser:
...
 
-We are using `kramdown` markdown converter. You can find out more about it here [kramdown](http://kramdown.gettalong.org). We highly recommend going through its syntax documentation 
-to get yourself more familiar with the coding style. 
+We are using `kramdown` markdown converter. You can find out more about it here [kramdown](http://kramdown.gettalong.org). We highly recommend going through its syntax documentation
+to get yourself more familiar with the coding style.
+
+#### Support for multiple authors
+
+Replace 'author' in the blog post with an array of authors like this:
+```yml
```
Comment (btiernay) [inline README.md:91]: 👍

File: _layouts/category.html [hunk 7]
```diff
@@ -17,57 +18,65 @@
...
+                <div class="small-12 columns b30 blog_teaser">
+                  <span class="date-display"><strong>{{ post.date | date: "%b. %-d, %Y" }}</strong></span>
+                  <h3 class="post-title"><a href="{{ site.url }}/{{ post.url }}">{{ post.title }}</a></h3>
+                    <p class="post-author">By:
+                    {% for currentAuthorId in authors %}
+                      {% assign author = site.data.authors[currentAuthorId] %}
+                      <a href="{{ site.baseurl }}/blog/category/{{ currentAuthorId }}"><strong>{{ author.name }}</strong></a>{% unless forloop.last %}, {% endunless %}
+                    {% endfor %}
                     </p>
+                  <div class="circles">
```
Comment (btiernay) [inline _layouts/category.html:40]: Love the circles

File: _posts/vitalii_slobodianyk/2016-11-25-elasticsearch-shading.md [hunk 8]
```diff
@@ -237,7 +237,7 @@ There is no other configuration as all the dependencies relocations are defined
...
+There is a problem related to testing with Elasticsearch. Sometimes it is preferable to be able to query against a running instance of Elasticsearch in unit tests, because mocking it too cumbersome or even not possible. There are cases (like development of your own [domain-specific language](https://github.com/icgc-dcc/dcc-portal/tree/develop/dcc-portal-pql)), when it is not suitable to query against a real Elasticsearch instance and you need an `in-memory` one. There are not good 3rd party projects yet which provide this functionality for Elasticsearch 5.x. The only way is to use the provided [ESIntegTestCase](https://www.elastic.co/guide/en/elasticsearch/reference/5.0/integration-tests.html) even though it is [not recommended](https://discuss.elastic.co/t/5-0-0-using-painless-in-esintegtestcase/64447/12).
```
Comment (btiernay) [inline _posts/vitalii_slobodianyk/2016-11-25-elasticsearch-shading.md:240]: What changed here?

Comment (Fgerthoffert) [reply _posts/vitalii_slobodianyk/2016-11-25-elasticsearch-shading.md:240]: nothing, thanks Atom.

**Review by shanewilson — APPROVED** [review]

**Pull request merged**